starport/services/network/network.go

```diff
@@ -2,273 +2,78 @@ package network
 
 import (
 	"context"
-	"os"
 
-	"github.com/go-git/go-git/v5"
-	"github.com/go-git/go-git/v5/plumbing"
 	launchtypes "github.com/tendermint/spn/x/launch/types"
-	"github.com/tendermint/starport/starport/pkg/chaincmd"
 	"github.com/tendermint/starport/starport/pkg/cosmosaccount"
 	"github.com/tendermint/starport/starport/pkg/cosmosclient"
 	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
 )
 
-const (
-	SPNAddressPrefix = "spn"
-	gentxFilename    = "gentx.json"
-)
-
-// Builder is network builder.
-type Builder struct {
+// Network is network builder.
+type Network struct {
```

Comment:
I suppose it is little bit confusing. Comment says it is network builder but there is another network builder in cmd/network.go. I would consider picking another name for this struct or for that one in cmd/network.go

Reply:
names are pretty tough choose 🤣. this seems like not a blocker for this PR, but please feel free to suggest some names so we can handle this in another PR.